### PR TITLE
Add translations and solutions

### PR DESCRIPTION
**Pull request by majamanojlovic**
Korean

### CODE REVIEW

**Comment by majamanojlovic** [comment]
Hi @martinohanlon, could you please take a look at this project? It said that there is one issue when I was setting up the integration in Crowdin so I just want to make sure everything's ok. Thanks!

File: ko-KR/step_1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
++ 스프라이트에 선택한 소리가 있음을 확인할 수 있을 겁니다.
+
+![스프라이트에 대한 새로운 소리](images/sound-imported.png)
+
+커뮤니티 기여 번역
+
+이 프로젝트는 강태원 가 번역하였고 홍주희 가 검토하였습니다.
+
+우리의 놀라운 번역 자원 봉사자들은 전 세계의 어린이들에게 코딩을 배울 수 있는 기회를 제공합니다. 우리 프로젝트 번역에 참여하여 더 많은 아이들에게 도움을 줄 수 있습니다. 자세한 내용은 [rpf.io/translators](https://rpf.io/translators)에서 확인하세요.
```
Comment (martinohanlon) [inline ko-KR/step_1.md]: @majamanojlovic should it have the phrase :

Our amazing translation volunteers give children around the world the opportunity to learn coding. We can help more children by participating in our project translation. For more information, please visit [rpf.io/translators] (https://rpf.io/translators)

... at the end of the ingredient?

**Comment by martinohanlon** [comment]
@majamanojlovic I made one comment https://github.com/raspberrypilearning/generic-scratch3-sound-from-library/pull/4/commits/90834f561d5ae0faf14509de46dcb5374fe1547c#r275888821

Other than that it looks fine to me.

File: ko-KR/step_1.md (hunk 1 above)
Comment (majamanojlovic) [reply ko-KR/step_1.md]: @martinohanlon yes, that's line 21.

Comment (martinohanlon) [reply ko-KR/step_1.md]: Yes. I was asking should it be there?

Comment (martinohanlon) [reply ko-KR/step_1.md]: If it should, feel free to merge.

Comment (majamanojlovic) [reply ko-KR/step_1.md]: Hmm well Nina said it should be at the end but now that you mention it she probably meant only projects, not ingredients. I'll remove it.

Comment (martinohanlon) [reply ko-KR/step_1.md]: This line above also says "This project was translated by Kang Tae Won and reviewed by Hong Ju-hee."

**Pull request merged**